pycqed/measurement/detector_functions.py

```diff
@@ -1359,54 +1359,84 @@ def process_data(self, data_raw):
             self.state_prob_mtx_list = state_prob_mtx_list
 
             classified_data = self.classify_shots(
-                data, self.classifier_params_list, self.state_prob_mtx_list,
-                self.get_values_function_kwargs.get('averaged', True),
-                self.get_values_function_kwargs.get('thresholded', True))
+                data, classifier_params_list=self.classifier_params_list,
+                state_prob_mtx_list=self.state_prob_mtx_list,
+                **self.get_values_function_kwargs)
 
         return classified_data.T
 
     def classify_shots(self, data, classifier_params_list,
-                       state_prob_mtx_list=None, averaged=False,
-                       thresholded=True):
+                       averaged=True, thresholded=False,
+                       ro_corrected_stored_mtx=False,
+                       ro_corrected_seq_cal_mtx=False,
+                       state_prob_mtx_list=None, **kw):
```

Comment:
Could you add a docstring explaining the meaning of all the parameters of this function.

Reply:
I now added explicit docstring.